cusfsim/case.py

```diff
@@ -81,7 +114,7 @@ class Dimension(PFDataStructs.Dimension):
     >>> repr(d)
     'cusfsim.case.Dimension(0, 1, -2, 0, 0, 0, 0)'
 
-    It supports indexing and the sequence property
+    The class also supports indexing and the sequence property
```

Comment:
I've just noticed that this is a thinko. It should be "and the sequence protocol". As it is, I'll fix in a separate PR.